### PR TITLE
[PFSP2-0005] Terms of Service

### LINKED ISSUE #62: [PFSP2-0005] Terms Of Service
### Priority 
- [x] Low
- [ ] Mid
- [ ] High

### Links
Name | Link
-|-
User Story | [link]()
Figma | [link](https://www.figma.com/file/YWJimHMl9fq3v4dQBZSuQo/Hi-Fi-User?node-id=393%3A5627)

### Completion
Expected | Time
-|-
Expected Start | 1-09-2020
Expected End | 29-09-2020

image

### PR DESCRIPTION
**Pull request by SetyaG**
Closes #62 
***
## Description
Create the Terms of Service page

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [link](#62)

## Todos
- [X] UI
- [ ] Coordinator
- [ ] Network
- [X] Tests
- [ ] Documentation

## Screenshot / Video
Screenshot consisting of changes made on this branch.

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)
Screen Shot 2020-09-21 at 12 21 18

## Notes
Additional notes that you have either technical debt, blocker, etc.

### CODE REVIEW

File: MainApp/MainApp/Feature/Profile/Terms of Service/TermsOfServiceController.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//
+
+import UIKit
+import Component
+
+class TermsOfServiceViewController: MasterViewController, TermsOfServiceViewProtocol {
+    
+    // MARK: - Outlet
+    @IBOutlet weak var TOSScrollView: UIScrollView!
+    @IBOutlet weak var TOSLabel: UILabel! {
```
Comment (inxudianz) [inline MainApp/MainApp/Feature/Profile/Terms of Service/TermsOfServiceController.swift]: tos need to be lowercased

**Review by inxudianz — CHANGES_REQUESTED** [review]
fix unit test as well as upload gif

File: MainApp/MainApp/Feature/Profile/Terms of Service/TermsOfServiceController.swift (hunk 1 above)
Comment (SetyaG) [reply MainApp/MainApp/Feature/Profile/Terms of Service/TermsOfServiceController.swift]: done

**Comment by SetyaG** [comment]
terms of service

**Comment by SetyaG** [comment]
Unit test fixed
Screen Shot 2020-09-23 at 17 20 25

**Pull request merged**